packages/flutter/test/widgets/inherited_test.dart

```diff
@@ -84,6 +84,37 @@ void main() {
     expect(log, equals(<TestInherited>[first, third]));
   });
 
+  testWidgets('InheritedElement hasListeners', (WidgetTester tester) async {
+    final GlobalKey key = GlobalKey();
+
+    await tester.pumpWidget(TestInherited(
+      key: key,
+      child: Container(),
+    ));
+
+    // ignore: avoid_as
+    final InheritedElement inheritedElement = key.currentContext as InheritedElement;
```

Comment:
I believe this should work without the "as InheritedElement", no?

If not: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores

Reply:
Done, is that good enough?

Reply:
@chunhtai  done